packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/snapshot_control_pane.dart

```diff
@@ -17,12 +20,77 @@ class SnapshotControlPane extends StatelessWidget {
   Widget build(BuildContext context) {
     return ValueListenableBuilder<bool>(
       valueListenable: controller.isProcessing,
-      builder: (_, isProcessing, __) => Row(
+      builder: (_, isProcessing, __) {
+        final current = controller.selectedItem as SnapshotListItem;
+
+        return Row(
+          children: [
+            const SizedBox(width: defaultSpacing),
+            if (!isProcessing && current.heap != null) ...[
+              _DiffDropdown(
+                current: current,
+                list: controller.snapshots,
```

Comment:
a nice polish could be to remove current from the diff drop down list, so we only show things you're interested in diffing against (no point in diffing current against current)

Reply:
I use 'current' as place holder for the option to not diff the snapshot at all. And I replace the name of snapshot wit '-':

I think it creates nice visual mapping between list at the left and list in dropdown:

<img width="503" alt="Screen Shot 2022-09-21 at 4 33 42 PM" src="https://user-images.githubusercontent.com/12115586/191627960-e755d84d-cb93-448f-b4f3-dde8102509c8.png">